docs/docs/configuration.md

```diff
@@ -88,7 +88,9 @@ modules:
 <br>
 
 ####Configuring global build
-If you would like to run some builder process before running builders of the specific modules, define it in the `build-parameters` section at global level in mta.yaml as follows:
+
+If you would like to run some builder process before running builders of the specific modules, define it by using the `build-parameters` section at global level in mta.yaml as follows:
```

Comment:
If you would like to run a builder process before running the builders of the specific modules, define it by using the "build-parameters" section at the global level in the "mta.yaml" file as follows:

Reply:
I will change it this way:
If you would like to run additional build steps before running the builders of the specific modules, define it by using the build-parameters section at the global level in the mta.yaml file as follows:

Reply:
A minor correction below, alsot add back ticks:
...define them by using the `build-parameters` section at the global level in the `mta.yaml` file...